src/installer/test/Assets/TestProjects/BundleProbeTester/Program.cs

```diff
@@ -9,13 +9,17 @@ namespace BundleProbeTester
 {
     public static class Program
     {
+        // The return type on BundleProbeDelegate is byte instead of bool because
+        // using non-blitable bool type caused a failure (incorrect value) on linux-musl-x64.
+        // The bundle-probe callback is only called from native code in the product
+        // Therefore the type on this test is adjusted to circumvent the failure.
         [UnmanagedFunctionPointer(CallingConvention.StdCall)]
-        public delegate bool BundleProbeDelegate([MarshalAs(UnmanagedType.LPWStr)] string path, IntPtr size, IntPtr offset);
+        public delegate byte BundleProbeDelegate([MarshalAs(UnmanagedType.LPWStr)] string path, IntPtr size, IntPtr offset);
```

Comment:
Is this ok to be `LPWstr` or should become `IntPtr`? 
If the goal is to inline or avoid a stub, can that happen if the call needs to pin the string?

Reply:
Yes the `LPWstr` is OK.
I spoke with @jkoritzinsky  about this today, looks like the problem was wrt managed `bool` vs native `bool` values in the return type.  Please see: https://github.com/dotnet/runtime/issues/36372.